pkg/minishift/registration/utils.go

```diff
@@ -26,12 +26,18 @@ import (
 )
 
 type RegistrationParameters struct {
-	Username string
-	Password string
+	Username           string
+	Password           string
+	SkipRegistration   bool
+	SkipUnRegistration bool
 }
 
 // Register host VM
 func RegisterHostVM(host *host.Host, param *RegistrationParameters) error {
+	if param.SkipRegistration {
+		log.Debug("User choosed to skip registration")
```

Comment:
@LalatenduMohanty Why do you think this something part of console because to achieve that user need to use `--skip-registration` flag which imply he choosed to do it. If that was something implicit we were doing than it did make sense but right now not to me.